tests/nightly/model_backwards_compatibility_check/train_mxnet_legacy_models.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+#Author: Piyush Ghai
+
+run_models() {
+	echo '=========================='
+	echo "Running training files and preparing models"
+	echo '=========================='
+	python mnist_mlp_module_api_train.py
```

Comment:
what are the consequences of having one of these scripts crash with respect to the integrity of the generated data?

Reply:
I did not quite fully understand by what you meant by integrity here. 
Are you suggesting that I do a checksum validation on the generated models and other relevant files while downloading them in the inference files ?

Reply:
if one of the training script randomly crashes, let's say for dataset unavailability, what are the consequences of the fact that the data stored in s3 will be incomplete for that given run?

Reply:
That's gracefully handled in this commit : https://github.com/apache/incubator-mxnet/pull/11626/commits/7c41488fe3d8679ec12a5d0694a596cc6a72017d 

As of now, I'm exiting the inference script with a message indicating no models found in S3.